### ISSUE TITLE
WebM packaged content fails to decode in Chrome

### ISSUE THREAD

**Issue by joeyparrish** [google/shaka-packager#158]
Using latest sources (5edd290694ec1a339d12c1c563d535d150b6f10d) packaged WebM content fails to decode in Chrome.  The previous build I was using (fe6775a50978204db1f558f40a7add5f42aea3a6) worked.

I will perform a `git bisect` to try to track down the relevant change.

**Comment by joeyparrish** [comment]
Specific error from chrome:media-internals: `DecryptingAudioDecoder: failed to init decoder on decryptor`

Full log:

| timestamp | property | value |
| ----- | ----- | ----- |
| 00:00:00 00 | pipeline_state | kCreated |
| 00:00:00 00 | event | WEBMEDIAPLAYER_CREATED |
| 00:00:00 00 | url | blob:http://localhost/399de490-e6f0-43c4-9b91-268ee47b255e |
| 00:00:00 00 | pipeline_state | kInitDemuxer |
| 00:00:03 331 | duration | 60.021331 |
| 00:00:03 387 | found_video_stream | true |
| 00:00:03 387 | video_codec_name | vp9 |
| 00:00:03 387 | found_audio_stream | true |
| 00:00:03 387 | audio_codec_name | vorbis |
| 00:00:03 389 | pipeline_state | kInitRenderer |
| 00:00:03 401 | debug | DecryptingAudioDecoder: failed to init decoder on decryptor |
| 00:00:03 401 | audio_dds | true |
| 00:00:03 401 | audio_decoder | FFmpegAudioDecoder |
| 00:00:03 402 | debug | DecryptingVideoDecoder: failed to init decoder on decryptor |
| 00:00:03 402 | video_dds | true |
| 00:00:03 402 | video_decoder | VpxVideoDecoder |
| 00:00:03 402 | pipeline_state | kPlaying |
| 00:00:03 442 | event | PLAY |
| 00:00:11 557 | height | 576 |
| 00:00:11 557 | width | 768 |
| 00:00:15 375 | error | video decode error |
| 00:00:15 424 | pipeline_error | pipeline: decode error |
| 00:00:15 424 | event | PAUSE |
| 00:00:15 424 | pipeline_state | kStopping |
| 00:00:15 426 | pipeline_state | kStopped |
| 00:00:33 823 | event | WEBMEDIAPLAYER_DESTROYED |
```

**Comment by joeyparrish** [comment]
Broken in db92c9a22b46d6fbe5e00f1e3a0ab9117aa59724

**Comment by joeyparrish** [comment]
Confirmed that adding --webm_subsample_encryption=false fixes the issue.

**Comment by kqyang** [comment]
Thanks for checking. We enabled subsample encryption by default for VP9 a few weeks ago.

Chromium code has been updated to support WebM subsample encryption (http://crbug.com/630344), which will be included in Chrome M55.

Here is the PSA announcement for WebM subsample support in Chrome: https://groups.google.com/a/chromium.org/forum/#!searchin/chromium-dev/subsample/chromium-dev/HvGava_WUjE/N9BDDgbYCQAJ
